task02.rb

```diff
@@ -0,0 +1,28 @@
+#2. For a given array that contains some strings output the content of those strings separated with whitespace between them, having capitalized first word and ending with ".", e.g. [[" thIs ", "Is"], nil, " My  ", ["   first"], "", :program] should output "This is my first program."
+
+def task02(arr)
+  if arr.class != Array
+    raise "The input type is invalid, expect an array"
```

Comment:
@MaraEfimova In general I wanted some sort of error, e.g. `raise ArgumentError, "Having error class is better"`. It is still throwing a RuntimeError, but we should always use an explicit error class.